operator/controller/src/main/resources/k8s/examples/kafkasql/plain/ephemeral.kafka.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: kafka.strimzi.io/v1beta2
```

Comment:
Should those `examples` resources stay in the `test` scope?
Do we need them at runtime in the Operator?

Reply:
Not necessarily in runtime, but my thinking is that we can use those examples in documentation for example, while making sure they are tested. If you have a suggestion on how to better organize this, let me know.

Reply:
Consider the `test/resources` folder, so that they are not going to be bundled with the final JARs